.github/workflows/other-tests.yml

```diff
@@ -389,6 +389,24 @@ jobs:
               cd e2e/bug-9574
               cd test
               ../../../phpstan
+          - php-version: 8.2 # same test as below but different order of commands
+            ini-values: memory_limit=256M
+            operating-system: ubuntu-latest
+            script: |
+              cd e2e/bug8778
+              composer dump
+              ../../phpstan clear-result-cache -q
+              ../../phpstan analyse -q
+              ../../phpstan analyse -vvv --generate-baseline --error-format=failWithoutResultCache
```

Comment:
I just checked AnalyseCommand and the codepath with `--generate-baseline` actually isn't influenced by the selected error formatter. So this isn't going to fail.

I really like `--error-format=failWithoutResultCache` and am looking forward to use it in other E2E tests, but not sure what to do about it here.

Reply:
I had no better idea either. I went back using the `grep` way for the `--generate-baseline` case.

to make it more robust, I am doing a initial run, which expects the error message beeing in stdout and the subsequent run without the very same string.

hopefully thats good enough.

Reply:
Sleeping over it I got a different idea. 

What about adding a new cli option, e.g `--assert-result-cached` or `--fail-when-no-result-cache` instead of using a error-formatter?

Reply:
implemented a ENV var approach in https://github.com/phpstan/phpstan-src/pull/2611